post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc

```diff
@@ -8,26 +12,48 @@ toc::[]
 
 == Scheduling workloads on clusters with multi-architecture compute machines
 
+endif::openshift-rosa,openshift-rosa-hcp[]
+
+//For ROSA builds
+ifdef::openshift-rosa,openshift-rosa-hcp[]
+:context: multi-architecture-compute-scheduling
+[id="multi-architecture-compute-scheduling"]
+= Scheduling workloads on clusters with multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+[IMPORTANT]
+====
+Multi-architecture compute machines are supported only for {product-title} with {hcp}.
+====
+endif::openshift-rosa,openshift-rosa-hcp[]
+
 Deploying a workload on a cluster with compute nodes of different architectures requires attention and monitoring of your cluster. There might be further actions you need to take in order to successfully place pods in the nodes of your cluster.
```

Comment:
If you only need this paragraph from the assembly and you this is an urgent change,  you could turn this paragraph into a snippet. Pros of this approach:
* It would allow you to get this merged quickly 
* You (or another writer) wouldn't need to refactor the assembly to a module like it should be
* You wouldn't need the symlink, you could create a new assembly for ROSA/HCP and use the snippet as the abstract
* There would be no need to try and roll the dice on a conditionalized, double TOC (which I suspect will cause problems in Pv1)

Reply:
Thanks for the thoughtful and cautious review, Michael!

(1) This was initially lodged against 4.16 only because of the HCP migration aiming at 4.17+; now that that branch has been merged I can refactor this so that it can go into main and enterprise-4.17+ as usual.

(2) Absolutely happy to have Kathryn's input here - as far as I know, Pv1 will only receive the content from one condition at a time; the "double `toc`" won't be doubled by the time it gets there. That said, given the amount of this content that needs to be reviewed and re-verified for ROSA+HCP, I can see how a snippet might seem like a better approach. I'll review against what is in main right now and see what seems sensible.